ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -744,7 +744,26 @@ extension BrowserViewController: WKNavigationDelegate {
       let mimeType = response.mimeType.flatMap({ UTType(mimeType: $0) }),
       mimeTypesThatRequireSFSafariViewControllerHandling.contains(mimeType)
     {
-      handleLinkWithSafariViewController(url, tab: tab)
```

Comment:
This handles other mime types as well not just mobile configurations

Reply:
Hmm I think it should do it also for the calendar type. 
So both the calendar and the mobile config since you can do the same issue on both.

I'll have to change the text I guess, to handle both.

Reply:
Fixed. Added calendar support.